event_webhook_types.go

```diff
@@ -567,9 +569,13 @@ type MergeEvent struct {
 		Source                   *Repository  `json:"source"`
 		Target                   *Repository  `json:"target"`
 		HeadPipelineID           *int         `json:"head_pipeline_id"`
+		LastEditedAt             string       `json:"last_edited_at"`
+		LastEditedByID           int          `json:"last_edited_by_id"`
+		StateID                  StateID      `json:"state_id"`
```

Comment:
I prefer to use the order from the docs... Which means `LastEditedAt` and `LastEditedByID` should go above `MilestoneID` and `StateID` should go directly below `MilestoneID`...

Reply:
Fixed order